docs/monthly-meeting/2024-07.md

```diff
@@ -70,8 +70,8 @@ Please take a second to read through it!
   - [Ned] interested in the idea spreadsheet; maybe it can help even offline
   - [Carol] At last PyCon US, Shauna and Melanie tried to make their first docs
     PR, and found some issues in the onboarding process
```

Comment:
```suggestion
    PR, found some issues in the onboarding process, and documented them
    in the following two Google Docs:
```
This optionally goes with the next suggestion, in case we want to "warn" the users more explicitly that the links points to Google Docs.

Reply:
Thanks, the other change on its own is a good improvement.